src/SenseoWifi.cpp

```diff
@@ -14,6 +14,8 @@ Released under some license.
 #include "constants.h"
 #include "testIO.cpp"
 
+void ICACHE_RAM_ATTR ledChangedHandler();

```

Comment:
Is this line needed?

Reply:
No, you are right ... is removed in a following commit. (together with codestyle changes ;) )